bower.json

```diff
@@ -6,20 +6,23 @@
   "dependencies": {
     "angular": "1.4.7",
     "angular-animate": "1.4.7",
+    "angular-clipboard": "~1.1.1",
     "angular-cookies": "1.4.7",
     "angular-dragdrop": "1.0.9",
     "angular-filter": "~0.5.4",
     "angular-loading-bar": "0.7.1",
     "angular-material": "0.11.2",
+    "angular-ng-sortablejs": "~1.2.1",
     "angular-route": "1.4.7",
     "angular-sanitize": "1.4.7",
-    "angular-ui-sortable": "~0.13.4",
     "jquery": "2.1.4",
     "json3": "~3.2.4",
     "ng-file-upload": "~5.0.9",
     "ng-grid": "2.0.14",
     "ngFx": "~1.1.0",
-    "underscore": "1.8.3"
+    "underscore": "1.8.3",
+    "ckeditor": "~4.5.4",
+    "angular-ckeditor": "~1.0.0"
```

Comment:
ckeditor breaks design, if we are gonna have a rich text editor (which we dont need atm) we are gonna implement it using MD